README.md

```diff
@@ -3,24 +3,24 @@
 A free-to-use tool that scans container images for package vulnerabilities.
 
 ## Overview
-Aqua Security's MicroScanner lets you check your container images for vulnerabilities. If your image has any known high-severity issue, MicroScanner can fail the image build, making it easy to include as a step in your CI/CD pipeline. 
+Aqua Security's MicroScanner lets you check your container images for vulnerabilities. If your image has any known high-severity issue, MicroScanner can fail the image build, making it easy to include as a step in your CI/CD pipeline.
```

Comment:
See how there is a change to all the line endings? Check this out: https://help.github.com/articles/dealing-with-line-endings/#platform-windows